### PR TITLE
User auth

### PR DESCRIPTION
**Pull request by rinkishivani**

### CODE REVIEW

File: accounts/forms.py [hunk 1]
```diff
@@ -20,11 +20,27 @@ class Meta:
 class UserUpdateForm(forms.ModelForm):
     """created form for update user profile"""
 
-    phone_no = forms.RegexField(min_length=10, max_length=10, regex=r'^\d+$',
-                                error_messages={'invalid': 'Enter a valid integer.'})
-    email = forms.RegexField(regex= '^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$'
-                             ,
-                             error_messages={'invalid': 'Enter a valid email address.'})
+    first_name = forms.RegexField(max_length=30, regex='^[a-zA-Z._%+-]+$',
```
Comment (lalitvasoya) [inline accounts/forms.py]: Keep all regex in single constant file instead of define in the form field directly

Comment (rinkishivani) [reply accounts/forms.py]: Done

**Review by lalitvasoya — APPROVED** [review]

File: accounts/constant.py [hunk 2]
```diff
@@ -4,4 +4,4 @@ class FormRegex:
     EMAIL_REGEX = '^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$'
     ZIPCODE_REGEX = r'^[0-9]{6}$'
     NAME = '^[a-zA-Z]+$'
-    PHONE_NO = '^([987]{1})(\d{1})(\d{8})'
+    PHONE_NO = r'^\d{10}$'
```
Comment (lalitvasoya) [inline accounts/constant.py:7]: add const in class for a group

File: accounts/forms.py [hunk 3]
```diff
@@ -28,9 +28,9 @@ class UserUpdateForm(forms.ModelForm):
     """created form for update user profile"""
 
     first_name = forms.RegexField(
-        max_length=30,
+        max_length=25,
         regex=FormRegex.NAME,
-        error_messages={'invalid': 'Enter a valid integer.'},
+        error_messages={'invalid': 'Enter a valid name.'},
```
Comment (lalitvasoya) [inline accounts/forms.py]: add messages in message class and refer that class here

File: static/css/style.css [hunk 4]
```diff
@@ -2484,3 +2484,33 @@ h2 {
    input.error {
    border: 2px solid red;
 }
+
```
Comment (lalitvasoya) [inline static/css/style.css]: create new CSS file based on the requirement and add reference in HTML

**Review by lalitvasoya — CHANGES_REQUESTED** [review]

File: accounts/forms.py (hunk 3 above)
Comment (rinkishivani) [reply accounts/forms.py]: added messages file

File: static/css/style.css (hunk 4 above)
Comment (rinkishivani) [reply static/css/style.css]: created separate CSS file for profile, 404 page and coming soon image

**Review by lalitvasoya — APPROVED** [review]

**Pull request merged**